frontend/src/app/modules/settings/user/user-profile-page/user-profile-page.component.ts

```diff
@@ -111,29 +110,39 @@ export class UserProfilePageComponent extends BaseComponent implements OnInit {
                     this.changeCountryCode(this.userForm);
                 }
             });
+
+        this.settingPageService.updateButtonClickEvent$.pipe(this.untilThis).subscribe(() => {
+            this.OnSubmit();
+        });
```

Comment:
See the previous comment

Reply:
it's the same as <button (click)="OnSubmit()"> and works in sync.

Reply:
As I mentioned subscribe in subscribe is anti-petters